src/actions/lint.ts

```diff
@@ -1,15 +1,17 @@
-const configPrettier = require("../config/prettier.config");
-const configEslint = require("../config/eslint.config");
+import { DefineAction } from "eilos";
 
-function runPrettier(ctx, argv) {
+import { Options } from "../options";
+import configEslint from "../config/eslint.config";
+import configPrettier from "../config/prettier.config";
+import type { PresetRuntimeContext } from "../options";
+
+function runPrettier(ctx: PresetRuntimeContext, argv: string[]) {
   // Get file patterns to match from `options` or use the default pattern
-  let prettier_file_patterns;
-  const options = ctx.getConfig("options");
-  if (options.prettier_file_patterns && options.prettier_file_patterns.length) {
-    prettier_file_patterns = options.prettier_file_patterns;
-  } else {
-    prettier_file_patterns = ["src/**/*.js", "src/**/*.ts", "**/*.md"];
-  }
+  const prettier_file_patterns = ctx.getConfig("prettierFilePatterns", [
+    "src/**/*.js",
+    "src/**/*.ts",
+    "**/*.md",
```

Comment:
totally unrelated: i think it would be nice to run prettier on all files by default.

Reply:
Not sure it's a good idea, because this will automatically include `dist` directories and `node_modules` by default, no? 🤔